src/Raven.Client/Http/AggressiveCacheMode.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Raven.Client.Http
+{
+    public enum AggressiveCacheMode
+    {
+        Unknown = 0,
+        TrackChanges = 1,
+        TrackChangesAndDoBackgroundRefresh = 2, // TODO arek - NotImplementedException
```

Comment:
I don't like this. Background refresh can be _really_ expensive and we don't want to do things behind the user's back

Reply:
This will make things easier. Arek added this because it was written in the issue to implement this.

Reply:
Yeah, the idea of having the background update was taken from https://issues.hibernatingrhinos.com/issue/RavenDB-7498 but I had some doubts how to do it properly so I left it without the implementation.

I'll drop this option entirely then.

Reply:
Fixed in https://github.com/ravendb/ravendb/pull/9179